apps/meteor/app/livechat/imports/server/rest/rooms.ts

```diff
@@ -70,3 +71,34 @@ API.v1.addRoute(
 		},
 	},
 );
+
+API.v1.addRoute(
+	'livechat/rooms/:rid/priority',
+	{
+		authRequired: true,
+		validateParams: { POST: isPOSTLivechatRoomPriorityParams },
+		permissionsRequired: {
+			POST: { permissions: ['view-l-room'], operation: 'hasAny' },
+			DELETE: { permissions: ['view-l-room'], operation: 'hasAny' },
```

Comment:
Within the tests below you're using another permission named `manage-livechat-priorities`, however, here within the definitions, I don't see that permission. TBH, I'm not really sure about the permissions of this priority feature, so I wanted to confirm if this isn't an issue

Reply:
well, this feature is about prioritizing an existing room with existing priorities, so it's more of a managing rooms permission, not managing-livechat-priorities, but good point nevertheless.

Reply:
Oh, I got it now. Thanks for the explanation!